hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1447,6 +1447,17 @@ def alter_namespace(namespace_name, *args)
       @admin.modifyNamespace(nsb.build)
     end
 
+    #----------------------------------------------------------------------------------------------
+    # Get namespace's rsgroup
+    def get_namespace_rsgroup(namespace_name)
+      # Fail if namespace name is not a string
+      raise(ArgumentError, 'Namespace name must be of type String') unless namespace_name.is_a?(String)
+      nsd = @admin.getNamespaceDescriptor(namespace_name)
+      raise(ArgumentError, 'Namespace does not exist') unless nsd
+      res = nsd.getConfigurationValue("hbase.rsgroup.name")
+      res
```

Comment:
Can be replaced with single line:
```
nsd.getConfigurationValue("hbase.rsgroup.name")
```

Reply:
no, I tried, but it doesn't work。
like rsgroup_admin.rb 
  #--------------------------------------------------------------------------
    # get group of server
    def get_rsgroup_of_server(server)
      res = @admin.getRSGroup(
        org.apache.hadoop.hbase.net.Address.fromString(server)
      )
      raise(ArgumentError, 'Server has no group: ' + server) if res.nil?
      res
    end

Reply:
I meant this:
```
    def get_namespace_rsgroup(namespace_name)
      # Fail if namespace name is not a string
      raise(ArgumentError, 'Namespace name must be of type String') unless namespace_name.is_a?(String)
      nsd = @admin.getNamespaceDescriptor(namespace_name)
      raise(ArgumentError, 'Namespace does not exist') unless nsd
      nsd.getConfigurationValue("hbase.rsgroup.name")
    end
```

Reply:
yes, pr has updated， please review again！